### DISCUSSION TITLE
essay/2024/12/15/solo-preneur-conference

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#248] [Comments]
# essay/2024/12/15/solo-preneur-conference

지난 수요일에 마곡에서 조쉬님의 솔로프리너 1인 개발자 컨퍼런스에 연사로 참여하고 왔습니다. 400명 가까이 오셔서 무려 밤 10시가 될 때까지 계셨는데요. 질문이 150개가 넘었지만 시간이 늦어 답변을 거의 드리지 못했었습니다.

https://jeho.page/essay/2024/12/15/solo-preneur-conference.html

**Comment by erados** [comment]
감사합니다.

**Comment by kimtoma** [comment]
컨퍼런스에는 참석하지 못했는데 내용 잘 정리해서 공유해주셔서 감사합니다. :-)
